sdks/java/core/src/main/java/org/apache/beam/sdk/metrics/CounterCell.java

```diff
@@ -57,28 +57,11 @@ public Long getCumulative() {
     return value.get();
   }
 
-  @Override
-  public Counter getInterface() {
-    return this;
-  }
-
-  @Override
   public void inc() {
     add(1);
   }
 
-  @Override
   public void inc(long n) {
     add(n);
   }
-
-  @Override
-  public void dec() {
```

Comment:
They can be decremented. This is `CounterCell`, the way `Counter` interface was implemented (`DelegatingCounter`) was using this `CounterCell#inc` with positive/negative values for inc/dec respectively, `CounterCell#dec` was not used.
https://github.com/apache/beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/metrics/Metrics.java#L78-L78